db/migrate/20170627182248_drop_roles_if_exists.rb

```diff
@@ -0,0 +1,9 @@
+class DropRolesIfExists < ActiveRecord::Migration
+  def change
+    if table_exists?(:roles)
+      drop_table(:roles) do |t|
```

Comment:
I tested the rollback both on my local UIC and open databases--it effectively is a no-op (it doesn't restore the table, but also doesn't error).

This syntax doesn't make it super clear that that's what's happening, though--I can do separate up/down migrations to make it more obvious.